code.json

```diff
@@ -0,0 +1,41 @@
+[
```

Comment:
The [code.json validator](https://code.gov/#!/policy-guide/docs/compliance/inventory-code/tools/validate-schema) wants the square brackets removed. After that it, should be fully valid with version 1! :tada:

Reply:
Curious: do you know if the USGS prefers code.json version 1 over version 2? I should know these things, but I don't... I haven't had the pleasure of bringing projects into compliance yet.

Reply:
Thanks for catching that issue. I am not sure about the code.json version; I based this off the file in the ngwmn-ui project.

Reply:
array brackets removed